src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useCallback } from 'react';
```

Comment:
Please clean up all unused imports and rev locale files to fix merge conflicts before merging